### PR TITLE
Create Rule “know-the-basic-video-terminologies/rule”

### PR DESCRIPTION
**Pull request by SofieHong**
Automatically generated by Netlify CMS

### CODE REVIEW

**Comment by SofieHong** [comment]
@apforsyth  Hi Andrew！ Test please

File: rules/know-the-basic-terms-for-a-video/rule.md [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+
+### Primary footage (main video)
+
+Primary footage is the footage that "tells" the story. It's the main audio and video that often consists of one or more people discussing a topic or narrative. Primary footage is the predominant format for most documentaries, broadcasts, talk shows, or TV shows.
+
+For example, on the SSW TV YouTube channel, this video mainly consists of the primary footage.
+
+`youtube: https://www.youtube.com/embed/mQyzt4CV_Jw`
+
+**Figure: The [interview footage](https://youtu.be/mQyzt4CV_Jw?t=38) drives the story of the video**
```
Comment (tiagov8) [inline rules/know-the-basic-terms-for-a-video/rule.md:33]: Hey @SofieHong 

I think we should change all these "Figure: " to be "Video example: "...

1. Please action if you agree

File: rules/know-the-basic-terms-for-a-video/rule.md [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+  - title: Sofie Hong
+    url: https://www.ssw.com.au/people/sofie-hong
+related:
+  - use-cutaways
+created: 2022-10-31T12:40:29.280Z
+guid: 9a3bc559-98d6-4119-88b5-0b7f4f342f6b
+---
+Understanding basic video terminology allows for better communication in video creation. Using basic video terminology can make our video content more engaging. It's important and useful for anyone involved in video production to know the basic terminology.
+
+## Filming Terminology
```
Comment (tiagov8) [inline rules/know-the-basic-terms-for-a-video/rule.md:23]: 1. Change from "Terminology" to "Terminologies"

File: rules/know-the-basic-terms-for-a-video/rule.md [hunk 3]
```diff
@@ -0,0 +1,71 @@
+---
+type: rule
+title: Production -  Do you know the basic terms for a video?
```
Comment (tiagov8) [inline rules/know-the-basic-terms-for-a-video/rule.md:3]: I think the title should be "Do you know the basic video terminologies?"

1. Please action

"terms" might be confused with contracts

File: rules/know-the-basic-terms-for-a-video/rule.md [hunk 4]
```diff
@@ -0,0 +1,71 @@
...
+uri: know-the-basic-terms-for-a-video
+authors:
+  - title: Adam Cogan
+    url: https://www.ssw.com.au/people/adam-cogan
+  - title: Andrew Forsyth
+    url: https://ssw.com.au/people/andrew-forsyth
+  - title: Jonty Gardner
+    url: https://www.ssw.com.au/people/jonty-gardner
+  - title: Landon Maxwell
+    url: "https://www.ssw.com.au/people/landon-maxwell "
```
Comment (tiagov8) [inline rules/know-the-basic-terms-for-a-video/rule.md:13]: 1. Please remove quotes from the URL

File: rules/know-the-basic-terms-for-a-video/rule.md [hunk 5]
```diff
@@ -0,0 +1,71 @@
...
+
+### B-roll footage
+
+B-roll is used to visually support the primary video in telling the story. For example, if the primary footage talks about the office, the B-roll might be the employees, the team meeting etc.
+
+For example, when the main video/primary footage comes to “our developers are working really hard”, the B-roll footage is the developer at work.
+
+`youtube: https://www.youtube.com/embed/BioFgITYAFY`
+
+**Figure: [B-roll](https://youtu.be/BioFgITYAFY?t=63) of two developers discussing**
```
Comment (tiagov8) [inline rules/know-the-basic-terms-for-a-video/rule.md:45]: 1. Please change from "two" to "2"

As per https://ssw.com.au/rules/use-digits-instead-of-words

**Comment by tiagov8** [comment]
Hey @SofieHong 

Please have a look at my comments in your Markdown and action those changes ;)

File: rules/know-the-basic-terms-for-a-video/rule.md (hunk 5 above)
Comment (SofieHong) [reply rules/know-the-basic-terms-for-a-video/rule.md]: Hi Tiago,
  1. Please change from "two" to "2"
Done

File: rules/know-the-basic-terms-for-a-video/rule.md (hunk 4 above)
Comment (SofieHong) [reply rules/know-the-basic-terms-for-a-video/rule.md]: Done

File: rules/know-the-basic-terms-for-a-video/rule.md (hunk 3 above)
Comment (SofieHong) [reply rules/know-the-basic-terms-for-a-video/rule.md]: Done

File: rules/know-the-basic-terms-for-a-video/rule.md (hunk 2 above)
Comment (SofieHong) [reply rules/know-the-basic-terms-for-a-video/rule.md]: Done

File: rules/know-the-basic-terms-for-a-video/rule.md (hunk 1 above)
Comment (SofieHong) [reply rules/know-the-basic-terms-for-a-video/rule.md]: Done - I agree with the "Video example", it's more accurately

**Review by jontygardner — APPROVED** [review]
As per our conversation with Adam - publish this rule and work on the changes.

**Pull request merged**